siriuspy/siriuspy/devices/ids.py

```diff
@@ -1607,72 +1669,28 @@ def pitch(self):
         """Return ID pitch readback [mm]."""
         return self[self.PARAM_PVS.PITCH_OFFSET_RB]
 
-    @property
-    def pitch_mon(self):
-        """Return ID pitch monitor [mm]."""
-        return self[self.PARAM_PVS.PITCH_OFFSET_MON]
-
-    # --- center ---
-
-    @property
-    def center_mode_status(self):
-        """Return ID center mode status."""
-        return self[self.PARAM_PVS.CENTER_MODE_STS]
-
-    @property
-    def center_offset(self):
-        """Return ID center offset readback [mm]."""
-        return self[self.PARAM_PVS.CENTER_OFFSET_RB]
-
-    @property
-    def center_offset_mon(self):
-        """Return ID center offset monitor [mm]."""
-        return self[self.PARAM_PVS.CENTER_OFFSET_MON]
-
-    # --- set methods ---
-
-    def set_gap(self, gap, timeout=None):
-        """Set ID target gap for movement [mm]."""
-        return self.set_kparameter(gap, timeout)
-
-    def set_gap_speed(self, gap_speed, timeout=None):
-        """Set ID cruise gap speed for movement [mm/s]."""
-        return self.set_kparameter_speed(gap_speed, timeout)
-
-    def set_gap_speed_max(self, gap_speed_max, timeout=None):
-        """Set ID max cruise gap speed for movement [mm/s]."""
-        return self.set_kparameter_speed_max(gap_speed_max, timeout)
-
-    def set_taper(self, taper, timeout=None):
-        """Set ID target taper for movement [mm]."""
-        return self._write_sp(self.PARAM_PVS.KPARAM_TAPER_SP, taper, timeout)
-
+    # Set methods
     def set_pitch(self, pitch, timeout=None):
         """Set ID pitch for movement [mm]."""
         return self._write_sp(self.PARAM_PVS.PITCH_OFFSET_SP, pitch, timeout)
 
-    def set_center_offset(self, center_offset, timeout=None):
-        """Set ID center offset for movement [mm]."""
-        return self._write_sp(self.PARAM_PVS.CENTER_OFFSET_SP, center_offset, timeout)
-
     def set_center_mode(self, mode, timeout=None):
         """Set ID center mode True or False."""
-        if type(mode) != bool:
+        if type(mode) is not bool:
```

Comment:
using `isinstance` is more pythonic.
```suggestion
        if isinstance(mode, bool):
```